scripts/output/single/reportCEScalib.R

```diff
@@ -17,8 +17,8 @@ require(colorspace)
 require(gdx)
 require(grid)
 
-gdx_name     <- "fulldata.gdx"        # name of the gdx  
-gdx_ref_name <- "input_ref.gdx"       # name of the reference gdx (for policy cost calculation)
+gdx_name     <- "fulldata.gdx"             # name of the gdx
+gdx_ref_name <- "input_refpolicycost.gdx"  # name of the reference gdx (for policy cost calculation)
```

Comment:
why is here this gdx loaded?

Reply:
In the sense of "why do we need it here anyway?" or "should be input_ref.gdx" as before?

Was introduced here: https://github.com/remindmodel/remind/commit/70d1ebd2d6a7b5d34d7abf06b279521744e5bdd2

Reply:
in the sense "should be input_ref.gdx" as before. I wondered why we calculate policy costs in this script. And actually, no input_ref.gdx file is used in the script could be deleted.

Reply:
done